packages/preload-docker-extension/src/index.ts

```diff
@@ -85,10 +85,16 @@ export class DockerExtensionPreload {
       execOptions,
     );
     if (rawResult.error) {
-      throw new Error(rawResult.error);
+      const error: any = { toString: () => rawResult.stderr };
```

Comment:
Shouldn't this return the error as before? Maybe it failed before the process has been launched and stderr doesn't return anything

Reply:
yes, wrong copy paste

Reply:
code has been removed